src/ima-engine-decorator.js

```diff
@@ -1,5 +1,5 @@
 // @flow
-import {BaseEngineDecorator, FakeEvent} from '@playkit-js/playkit-js';
+import {BaseEngineDecorator, FakeEvent, Html5EventType} from '@playkit-js/playkit-js';
```

Comment:
`Html5EventType` is not exposed in playkit.js use `EventType`

Reply:
Of course it is look on [Playkit.js](https://github.com/kaltura/playkit-js/blob/master/src/playkit.js#L100)

Reply:
ok. it's new.
but in your usecase it's not correct because `AUTOPLAY_FAILED` it's not native HTML event